### PR TITLE
Shared common config files

### PR DESCRIPTION
**Pull request by lionelvillard**
Fixes #

## Proposed Changes

- Share KafkaSource config files when appropriate.
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: config/source/common/serviceaccount.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Knative Authors
+# Copyright 2019 The Knative Authors
```
Comment (matzew) [inline config/source/common/serviceaccount.yaml:1]: hrm.... ?

**Comment by matzew** [comment]
Can you change the mixed up / incorrect header dates ? 

perhaps bump all to 2021 ?

File: config/source/common/serviceaccount.yaml (hunk 1 above)
Comment (antoineco) [reply config/source/common/serviceaccount.yaml:1]: Looks like a Git glitch. It was probably another file that got wrongly considered as "renamed".

**Review by antoineco — APPROVED** [review]
/lgtm

**Comment by lionelvillard** [comment]
These are not new files, so the copyright should not be changed.

@antoineco thanks for approving! `/lgtm` has been deprecated in favor of GitHub approval process.

**Comment by antoineco** [comment]
Good to know!

Regarding the copyright headers, in theory the date should reflect the year the code was "published", which on GitHub should correspond to a "release". In practice, I've never seen anyone apply that correctly :)

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**